src/python/pants/bin/remote_pants_runner.py

```diff
@@ -53,24 +54,15 @@ def restore_tty_flags(self):
 
 
 class PailgunClientSignalHandler(SignalHandler):
-    def __init__(self, pailgun_client: NailgunClient, pid: int, timeout: float = 1):
-        self._pailgun_client = pailgun_client
-        self._timeout = timeout
+    def __init__(self, pid: int):
         self.pid = pid
         super().__init__(pantsd_instance=False)
 
     def _forward_signal_with_timeout(self, signum, signame):
```

Comment:
It would be good to rename this helper to `_forward_signal` since it no longer deals with timeouts.